src/reducers/tripReducer.ts

```diff
@@ -0,0 +1,40 @@
+export const FETCH_TRIPS_START = 'FETCH_TRIPS_START';
+export const FETCH_TRIPS_SUCCESS = 'FETCH_TRIPS_SUCCESS';
+export const FETCH_TRIPS_ERROR = 'FETCH_TRIPS_ERROR';
+
+export type tripActions = 'FETCH_TRIPS_START' | 'FETCH_TRIPS_SUCCESS' | 'FETCH_TRIPS_ERROR';
+
+export interface TripState {
+  trips?: any;
+  status: tripStatus;
+}
+
+export enum tripStatus {
+  loading = 'loading',
+  succsess = 'success',
+  error = 'error',
+}
```

Comment:
I think we're going to use this reducer to process newly created trips. i.e.:  we will have another state variable to track the status of the "create trip request".

I suggest we re-name this `status` to `getTripsStatus`. Then the create trip request will be `createTripStatus`.

Also, we're going to have many status variables in the app.

Let's create a folder under `src` called `types`. And a file called index.ts. 

In the file, let's do

```typescript
export enum requestStatus {
  loading = 'loading',
  succsess = 'success',
  error = 'error',
}
```

then we import this type and make `getTripsStatus` of type `requestStatus`